openwisp_users/base/models.py

```diff
@@ -97,6 +97,24 @@ def is_owner_of_any_organization(self):
                 return True
         return False
 
+    def selected_org(self, org_field):
+        if self.is_superuser:
+            cache_key = 'selected_{}_org'.format('superuser')
+        else:
+            cache_key = 'selected_{}_org'.format(self.pk)
+        selected_org = cache.get(cache_key)
+        if selected_org is not None:  # If cache is not deleted
+            if selected_org is False:
+                return None
+            return selected_org
+        org_queryset = org_field._queryset
+        if org_queryset.count() == 1:
+            selected_org = org_queryset[0]
+        else:
+            selected_org = False  # Tells selected org is None
```

Comment:
I don't get it, here we are returning `False`, above we are doing 
```
if selected_org is False:
 return None
```

do we want to return None? Can you please explain this? :smile:

Reply:
Thanks, I think it has been missed.